robottelo/constants/repos.py

```diff
@@ -12,7 +12,7 @@
 CUSTOM_RPM_SHA = 'https://fixtures.pulpproject.org/rpm-with-sha/'
 CUSTOM_RPM_SHA_512 = 'https://fixtures.pulpproject.org/rpm-with-sha-512/'
 FAKE_5_YUM_REPO = 'https://rplevka.fedorapeople.org/fakerepo01/'
-FAKE_YUM_DRPM_REPO = 'https://fixtures.pulpproject.org/drpm-signed/'
+FAKE_YUM_MISSING_REPO = 'https://fixtures.pulpproject.org/missing-signed/'
```

Comment:
This repos throws a 404 error. Is this repo intended to be missing? :)

Reply:
Yes, the drpm repo was removed from fixtures.pulpproject.org a while ago, but it has been still used in `Notifications` component (that's why I requested review from @pnovotny ) as missing/unsyncable repo. So thought I should rename it to what it really is so people don't confuse it with some real drpm repo.

Reply:
Exactly, the repo is intended to be missing/unsyncable for the notifications test.

Reply:
@vsedmik missing-signed name might confuse as its just not signed, what about naming it as `missing-repo` or something?

Reply:
@Gauravtalreja1 sounds good. :+1: 
It could be also `f'https://fixtures.pulpproject.org/{gen_string("alpha")}/'`, but let's keep it `missing-repo`.